tools/pkglint/lib/rules.ts

```diff
@@ -376,11 +376,11 @@ export class NodeCompatibility extends ValidationRule {
 
   public validate(pkg: PackageJson): void {
     const atTypesNode = pkg.getDevDependency('@types/node');
-    if (atTypesNode && !atTypesNode.startsWith('^10.')) {
+    if (atTypesNode && !atTypesNode.startsWith('^12.')) {
       pkg.report({
         ruleName: this.name,
         message: `packages must support node version 10 and up, but ${atTypesNode} is declared`,
```

Comment:
```suggestion
        message: `packages must support node version 12 and up, but ${atTypesNode} is declared`,
```

Reply:
Good catch. `yarn.lock` is updated now.

Reply:
Did you include your changes in that most recent push? I don't see this rule or the yarn.lock updated.

Reply:
You are right. Forgot to add files to commit.